### ISSUE TITLE
First run asks to run with --config option which doesn't work

### ISSUE THREAD

**Issue by jjlharrison** [ghost2109/ash#2]
```
~ $ ash
run ash with --config option
~ $ ash --config
None
~ $
```

When first running ash it prompt to run with the `--config` option. Then when running this option it simply outputs "None" and no `~/.ash/ash.json` file is created.

**Comment by jjlharrison** [comment]
`--debug` option gives the following output:
```
~ $ ash --debug --config
Function __init__ returned an error: must be str, not intmust be str, not int
Error on line 56 TypeError must be str, not int
None
Function cmdloop returned an error: must be str, not intmust be str, not int
Error on line 56 TypeError must be str, not int
~ $
```

**Comment by ghost2109** [comment]
Fixed in version 1.2.6 removed --config option, it now check automatically .
